integrations/github/src/haystack_integrations/components/connectors/github/file_editor.py

```diff
@@ -0,0 +1,287 @@
+from base64 import b64decode, b64encode
```

Comment:
We should also add the license header text to all python files right?

Reply:
Done.